TestFoundation/TestJSONSerialization.swift

```diff
@@ -989,6 +989,7 @@ extension TestJSONSerialization {
             ("test_serialize_dictionaryWithDecimal", test_serialize_dictionaryWithDecimal),
             ("test_serializeDecimalNumberJSONObject", test_serializeDecimalNumberJSONObject),
             ("test_serializeSortedKeys", test_serializeSortedKeys),
+            ("test_colonPrettyPrintingMatchesDarwin", test_colonPrettyPrintingMatchesDarwin),
```

Comment:
Can you call this `test_serializePrettyPrinted()` please? In time it should be expanded to test more than just the colon positioning, plus it is not actually testing Darwin compatibility, rather a string which _currently_ matches Darwin. In other words, if Darwin is changed this test will not automatically start failing.